qutebrowser/browser/webengine/notification.py

```diff
@@ -361,13 +361,22 @@ def _on_adapter_clicked(self, notification_id: int) -> None:
             return
 
         try:
+            self._focus_first_matching_tab(notification)
```

Comment:
This should be above the `try:`, as the `except RuntimeError:` shouldn't be needed for this.